[xenial] fix `Unknown status message: u'IMPORT_OK'` related test failures

## Description

Unit tests involving importing an armored key/decryption appear to be failing, possibly related to #4013 (different error message though)

## Steps to Reproduce

`make test-xenial`

## Expected Behavior

All is merry and well

## Actual Behavior

A bunch of integration test failures, e.g.:

```
_____________________________ test_submit_message ______________________________

source_app = <Flask 'source_app'>, journalist_app = <Flask 'journalist_app'>
test_journo = {'id': 1, 'journalist': <Journalist washboard confront imitation demise affidavit elf freewill radish>, 'otp_secret': 'YKZ3S3CWXEALHRTL', 'password': 'exclusion transpire emission fragrant confusion swept sappy yelp', ...}

    def test_submit_message(source_app, journalist_app, test_journo):
        """When a source creates an account, test that a new entry appears
        in the journalist interface"""
        test_msg = "This is a test message."
    
        with source_app.test_client() as app:
            app.get('/generate')
            app.post('/create', follow_redirects=True)
            filesystem_id = g.filesystem_id
            # redirected to submission form
            resp = app.post('/submit', data=dict(
                msg=test_msg,
                fh=(StringIO(''), ''),
            ), follow_redirects=True)
            assert resp.status_code == 200
            app.get('/logout')
    
        # Request the Journalist Interface index
        with journalist_app.test_client() as app:
            _login_user(app, test_journo)
            resp = app.get('/')
            assert resp.status_code == 200
            text = resp.data.decode('utf-8')
            assert "Sources" in text
            soup = BeautifulSoup(text, 'html.parser')
    
            # The source should have a "download unread" link that
            # says "1 unread"
            col = soup.select('ul#cols > li')[0]
            unread_span = col.select('span.unread a')[0]
            assert "1 unread" in unread_span.get_text()
    
            col_url = soup.select('ul#cols > li a')[0]['href']
            resp = app.get(col_url)
            assert resp.status_code == 200
            text = resp.data.decode('utf-8')
            soup = BeautifulSoup(text, 'html.parser')
            submission_url = soup.select('ul#submissions li a')[0]['href']
            assert "-msg" in submission_url
            span = soup.select('ul#submissions li span.info span')[0]
            assert re.compile('\d+ bytes').match(span['title'])
    
            resp = app.get(submission_url)
            assert resp.status_code == 200
            decrypted_data = journalist_app.crypto_util.gpg.decrypt(resp.data)
>           assert decrypted_data.ok
E           assert False
E            +  where False = <gnupg._parsers.Crypt object at 0x7f7dbf4d2f10>.ok

tests/test_integration.py:83: AssertionError
---------------------------- Captured stdout setup -----------------------------
seq  name             file                                                      
---  ---------------  ----------------------------------------------------------
0    main             /tmp/pytest-of-www-data/pytest-0/test_submit_message0/data
----------------------------- Captured stderr call -----------------------------
2019-01-16 04:31:25,916 INFO generating key, entropy: 8192
2019-01-16 04:31:26,230 INFO 'washboard confront imitation demise affidavit elf freewill radish' logged in with the token 619296
Exception in thread Thread-421:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/local/lib/python2.7/dist-packages/gnupg/_meta.py", line 650, in _read_response
    result._handle_status(keyword, value)
  File "/usr/local/lib/python2.7/dist-packages/gnupg/_parsers.py", line 995, in _handle_status
    raise ValueError("Unknown status message: %r" % key)
ValueError: Unknown status message: u'IMPORT_OK'

Exception in thread Thread-430:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/local/lib/python2.7/dist-packages/gnupg/_meta.py", line 650, in _read_response
    result._handle_status(keyword, value)
  File "/usr/local/lib/python2.7/dist-packages/gnupg/_parsers.py", line 1757, in _handle_status
    super(Crypt, self)._handle_status(key, value)
  File "/usr/local/lib/python2.7/dist-packages/gnupg/_parsers.py", line 1656, in _handle_status
    raise ValueError("Unknown status message: %r" % key)
ValueError: Unknown status message: u'IMPORT_OK'
```

## Comments

I would bump to the latest python-gnupg and see if the issue was resolved upstream (I didn't see anything about this issue in the bugtracker, but worth a try), ref: #3622

--- a/securedrop/tests/conftest.py
+++ b/securedrop/tests/conftest.py
@@ -77,7 +77,7 @@ def config(tmpdir):
     tmp = data.mkdir('tmp')
     sqlite = data.join('db.sqlite')
 
-    gpg = gnupg.GPG(homedir=str(keys))
+    gpg = gnupg.GPG('gpg2', homedir=str(keys))
     for ext in ['sec', 'pub']:
         with io.open(path.join(path.dirname(__file__),
                                'files',

--- a/securedrop/tests/test_integration.py
+++ b/securedrop/tests/test_integration.py
@@ -411,7 +411,7 @@ def _can_decrypt_with_key(journalist_app, msg, key_fpr, passphrase=None):
     # to use it to test whether a message is decryptable with a specific
     # key.
     gpg_tmp_dir = tempfile.mkdtemp()
-    gpg = gnupg.GPG(homedir=gpg_tmp_dir)
+    gpg = gnupg.GPG('gpg2', homedir=gpg_tmp_dir)
 
     # Export the key of interest from the application's keyring
     pubkey = journalist_app.crypto_util.gpg.export_keys(key_fpr)

--- a/securedrop/tests/utils/env.py
+++ b/securedrop/tests/utils/env.py
@@ -37,7 +37,7 @@ def init_gpg():
     """Initialize the GPG keyring and import the journalist key for
     testing.
     """
-    gpg = gnupg.GPG(homedir=config.GPG_KEY_DIR)
+    gpg = gnupg.GPG('gpg2', homedir=config.GPG_KEY_DIR)
     # Faster to import a pre-generated key than to gen a new one every time.
     for keyfile in (join(FILES_DIR, "test_journalist_key.pub"),
                     join(FILES_DIR, "test_journalist_key.sec")):

